packages/edit-site/src/store/selectors.js

```diff
@@ -321,3 +321,27 @@ export function isNavigationOpened() {
 		version: '6.4',
 	} );
 }
+
+/**
+ * Whether or not the editor has a page loaded into it.
+ *
+ * @see setPage
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {boolean} Whether or not the editor has a page loaded into it.
+ */
+export function isPage( state ) {
```

Comment:
Should this be called `isPost` instead? It's already possible to edit a post from the site editor if it's linked to from the Navigation. Same for `hasPageContentLock` and `setHasPageContentLock` I guess 😄

Reply:
Yeah agree. Thinking I'll rename everything I'm adding in this PR to be `post` instead of `page` which better matches the WordPress convention of calling every CPT-like thing a _post_. I'll do it in a follow-up though 😅 (It's in my list in the PR's description.)